### DISCUSSION TITLE
How Does **kwargs work?

### DISCUSSION THREAD

**Discussion by wass-cloud** [pybamm-team/PyBaMM#3270] [Q&A] [answered]
When I try to run the following code, the linewidth does not change. 

pybamm.plot_summary_variables(sim_solved,["Capacity [A.h]",
                                          "Loss of lithium inventory [%]",
                                ],
                              linewidth=4                               
)

**Comment by brosaplanella** [comment]
I checked the code and I am not sure that this is working as QuickPlot seems to not take any kwargs. @tinosulzer should we turn this into an issue?

**Reply by valentinsulzer** [reply, marked as answer]
The kwargs for `plot_summary_variables` are only used for creating the figure in `plt.subplots`.
If you want to customize the plot you can copy the code from `plot_summary_variables` and edit it as you wish
